shallow_backup/printing.py

```diff
@@ -83,23 +84,32 @@ def print_section_header(title, color):
 
 
 def print_pkg_mgr_backup(mgr):
-	print("{}Backing up {}{}{}{}{} packages list...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW, mgr, Fore.BLUE,
-	                                                          Style.NORMAL, Style.RESET_ALL))
+	print("{}Backing up {}{}{}{}{} packages list...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW, mgr,
+															  Fore.BLUE, Style.NORMAL, Style.RESET_ALL))
 
 
 def print_pkg_mgr_reinstall(mgr):
-	print("{}Reinstalling {}{}{}{}{}...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW, mgr, Fore.BLUE, Style.NORMAL, Style.RESET_ALL))
+	print("{}Reinstalling {}{}{}{}{}...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW,
+												  mgr, Fore.BLUE, Style.NORMAL, Style.RESET_ALL))
 
 
-def prompt_yes_no(message, color):
+def print_pkg_mgr_error(cmd):
+	print("{}An error occurred while running:{} $ {}".format(Fore.RED, Style.RESET_ALL, cmd))
```

Comment:
Let's keep the error styling consistent and use `print_path_red()` to get the full error printing in red and the command not bolded (everything including, and past the `$`, should be in normal style). I realize this function is not that well named, but don't have any better suggestions.

Reply:
like this?
```
print("{}$ {}".format(print_path_red('An error occurred while running: ') cmd))
```

Reply:
```suggestion
	print_path_red("An error occurred while running: $", cmd)
```

Reply:
oh I'm an idiot

Reply:
wait did you want to just call `print_path_red()` from utils.py then and delete `print_shell_cmd_error()`?

Reply:
That wasn't what I was saying but that actually that makes more sense. Let's do that.